src/m365/graph/commands/subscription/subscription-add.ts

```diff
@@ -172,7 +178,8 @@ class GraphSubscriptionAddCommand extends GraphCommand {
       notificationUrl: args.options.notificationUrl,
       expirationDateTime: await this.getExpirationDateTimeOrDefault(logger, args),
       clientState: args.options.clientState,
-      includeResourceData: args.options.includeResourceData,
+      includeResourceData: args.options.withResourceData || args.options.includeResourceData,
+      withResourceData: args.options.withResourceData || args.options.includeResourceData,
```

Comment:
I don't think that will work. the `includeResourceData` is an option we may set over the API and the `withResourceData` is our CLI for Microsoft 365 name for this option now